tox.ini

```diff
@@ -31,6 +33,9 @@ deps =
     {[testenv:unittests]deps}
     coverage==3.7.1
     pytest-cov==1.8.1
+    pytest-capturelog==0.7
+    pytest-qt==1.3.0
+    pytest-mock==0.4.2
```

Comment:
Those are all already included by the `{[testenv:unittests]deps}` above, so shouldn't be added here again or pip will complain about multiple requirements.